impexp-core/src/main/java/org/citydb/core/operation/exporter/database/content/DBTunnelHollowSpace.java

```diff
@@ -119,61 +122,76 @@ public DBTunnelHollowSpace(Connection connection, CityGMLExportManager exporter)
 				}
 				surfaceADEHookTables = addJoinsToADEHookTables(TableEnum.THEMATIC_SURFACE, thematicSurface);
 			}
-			if (projectionFilter.containsProperty("interiorFurniture", tunnelModule)) {
-				CombinedProjectionFilter tunnelFurnitureProjectionFilter = exporter.getCombinedProjectionFilter(TableEnum.TUNNEL_FURNITURE.getName());
+			if (projectionFilter.containsProperty("hollowSpaceInstallation", tunnelModule)) {
 				Table tunnelFurniture = new Table(TableEnum.TUNNEL_FURNITURE.getName(), schema);
-				tunnelFurnitureExporter.addProjection(select, tunnelFurniture, tunnelFurnitureProjectionFilter, "tf")
-						.addJoin(JoinFactory.left(tunnelFurniture, "tunnel_hollow_space_id", ComparisonName.EQUAL_TO, table.getColumn("id")));
-				tunnelFurnitureADEHookTables = addJoinsToADEHookTables(TableEnum.TUNNEL_FURNITURE, tunnelFurniture);
+				select.addProjection(new ColumnExpression(new Select()
+						.addProjection(tunnelFurniture.getColumn("id"))
+						.addSelection(ComparisonFactory.equalTo(tunnelFurniture.getColumn("tunnel_hollow_space_id"), table.getColumn("id")))
+						.withFetch(new FetchToken(1)), "tfid"));
 			}
-			if (projectionFilter.containsProperty("hollowSpaceInstallation", tunnelModule)) {
+			if (projectionFilter.containsProperty("interiorFurniture", tunnelModule)) {
```

Comment:
Here should be "hollowSpaceInstallation" instead of "interiorFurniture"

Reply:
fixed with 5539d4066680714a24f854506a010ba7dfdc878d